web_poet/page_inputs/stats.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC
+from typing import Any
+
+
+class Stats(ABC):
+    """Stat collector."""
+
+    def __init__(self):
+        self._stats = {}
```

Comment:
Another thought, sorry :) May we should drop the concrete implementation, and only provide abc. The reason is that stats is a global state, so you always need the framework to provide it; Stats object should hold a reference to some global state, and change it in set/inc methods. It doesn't seem frameworks can reuse the implementation which we have here.